src/components/pick-list/pick-list.tsx

```diff
@@ -68,6 +69,13 @@ export class PickList<
    */
   @Prop({ mutable: true }) filteredItems: HTMLCalcitePickListItemElement[] = [];
 
+  /**
+   * **read-only** The currently filtered items
+   *
+   * @readonly
```

Comment:
@macandcheese Do you think `@readonly` could be picked up by the doc process? It'd prevent us from explicitly adding it to the JSDoc.